### ISSUE TITLE
Add support for delete/default declaration

### ISSUE THREAD

**Issue by ashleyj** [bytedeco/javacpp#80]
Add support for C++ default and delete declarations (http://www.stroustrup.com/C++11FAQ.html#default)

eg.
```
class X {
		// ...
		X& operator=(const X&) = delete;	// Disallow copying
		X(const X&) = delete;
	};
```
or explicit default:
```
class Y {
		// ...
		Y& operator=(const Y&) = default;	// default copy semantics
		Y(const Y&) = default;
	};
```

**Comment by saudet** [comment]
Should now work! Thanks for trying this stuff out :)

**Comment by saudet** [comment]
Released in version 1.2, thanks for reporting!
